packages/sdk/tests/test-utils.e2e.ts

```diff
@@ -0,0 +1,34 @@
+import { Wallet } from 'ethers';
+import { IExecAppModule, TeeFramework } from 'iexec';
+import { getWeb3Provider, Web3SignerProvider } from '../src/index.js';
```

Comment:
This import was problematic for unit tests. What it does is that it loads the entire SDK so we couldn't mock some modules *before* loading the SDK.

That's why I moved this to a `test-utils.e2e.ts` file that is only used in e2e tests.